Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/SystemCenter.tsx

```diff
@@ -28,7 +28,7 @@ import { BrowserRouter as Router, Route, NavLink, Switch } from 'react-router-do
 import queryString from "querystring";
 import { createBrowserHistory } from "history"
 import { SystemCenter } from './global';
-import { SystemCenter as SCTypes } from '@gpa-gemstone/application-typings';
+import { Application, SystemCenter as SCTypes } from '@gpa-gemstone/application-typings';
```

Comment:
you should be able to remove `as SCTypes` and use `SystemCenter.[...]` instead

Reply:
I have kept this as SCtypes as otherwise there is a naming error because of the functional component at the start of the file being named SystemCenter